src/api/cpp/nixl_descriptors.h

```diff
@@ -208,12 +208,11 @@ class nixlDescList {
                      const bool &sorted=false,
                      const int &init_size=0);
         /**
-         * @brief Deserializer constructor for nixlDescList from nixlSerDes object
-         *        which serializes/deserializes our classes into/from blobs
+         * @brief Deserializer constructor for nixlDescList from nixlDeserializer.
          *
-         * @param deserialize nixlSerDes object to construct nixlDescList
+         * @param deserialize nixlDeserializer object to construct nixlDescList
          */
-        nixlDescList(nixlSerDes* deserializer);
+        explicit nixlDescList(nixlDeserializer& deserializer);
```

Comment:
it is API change

@yosefe, @aranadive is it ok, or can only be done together with other API changes?